demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1119,6 +1120,11 @@ def add_to_list(self, object_type: IDSetType, obj):
 
         self._id_set_dict.setdefault(object_type, []).append(obj) if obj not in self._id_set_dict[object_type] else None
 
+    def add_to_dict(self, object_type: IDSetType, obj_name, obj_value):
```

Comment:
This name currently feels like it doesn't supply enough information about what this function actually does.
Let's see if we can improve iit

Reply:
done